packages/core/src/sdk/deliveryPromise/useDeliveryPromise.ts

```diff
@@ -366,6 +366,17 @@ export function useDeliveryPromise({
 
             return facet
           })
+          .sort((a, b) => {
+            // Define priority order: dynamic-estimate (0), shipping (1), delivery-options (2), others (3)
+            const getPriority = (key: string) => {
+              if (key === DYNAMIC_ESTIMATE_FACET_KEY) return 0
```

Comment:
I believe that after the changes in [#2958](https://github.com/vtex/faststore/pull/2958/files#diff-cba2d25d606c7eb475cfc6bf978d9e68c9808e1ef5404e1c3e80d090b3297405R388), we are splitting the DYNAMIC_ESTIMATE facet, so the won't need to add it here 🤔